### ISSUE TITLE
Feature request: Make vscode adoption easier for Visual Studio users

### ISSUE THREAD

**Issue by amervitz** [microsoft/vscode#104532]
Hi,

I'm a regular Visual Studio user, and I'm finding myself needing to use vscode more regularly. One thing that I'm finding detracts from feeling comfortable using it is subtle visual aesthetics and behaviors that might perhaps stem from vscode being an Electron based application, whereas Visual Studio is a Windows/WPF application.

Here are some examples of things that stand out to me. I could make a list much longer than this, but hopefully this gets the idea across. I may not be explaining some of these things with the best terminology since these are visual/gut feelings I have and I don't know how to express them properly or know the technical reasons for how some of these things work.

1. When hovering over files in the explorer, the mouse uses the hand icon like when hovering over links in a web page, but Visual Studio uses the normal pointer icon.
2. The tab for an open file is very big compared to Visual Studio, both in width, height, and font size.
3. The explorer, source control, extensions, etc. items are on the left side, whereas in Visual Studio they are either located on the right or accessible via other methods (status bars, file menus). I realize these can be moved to the right by a setting but it was something I had to figure out how to do.
5. Compared to Visual Studio, the file and context menus looks different. They have different options, no icons, and the name and shortcut are in a different order.
6. Even when I configure the same font and size in vscode as I use in Visual Studio, it doesn't look the same size. Maybe there is some difference in rendering technology behind this.
7. There isn't a toolbar in vscode, I don't think this is a feature.
8. Visual Studio feels like a desktop application with things being visually tighter and perceptively feeling snappier.

Aside from this list, which I could keep expanding upon if that is helpful, I feel like I'm using a web browser. I assume a lot of this is because it IS a web browser, but just because vscode is built upon web technologies, does it have to look and feel like a web page? I don't know how much of the user experience is restricted by the use of Electron.

If there are a lot of existing settings that already address some of these concerns to make vscode look and feel more like Visual Studio, a `Visual Studio emulation settings` option that sets that collection of settings with the necessary values all at once would be nice.

**Comment by misolori** [comment]
Hey, thanks for the feedback! We do try to be inspired by Visual Studio but also try to instill our own set of principles that make VS Code unique. They are different products who also target a diverse set of users, so that's why we have to diverge sometimes and create new paradigms. I'll try to respond to your feedback but know that at the core we are intentionally different:

1. We actually already have an issue that covers the hover cursor issue, i'd encourage you to upvote that one #55346. The biggest change here is that we don't have any hover feedback for icons so this makes it a bit trickier.
2. Our tabs are indeed bigger because our toolbar is also on the same line and we need more space to accommodate it. We 
  ![image](https://user-images.githubusercontent.com/35271042/90576584-8067db00-e173-11ea-9073-42dc1e2a76e4.png)
  - Depending on the theme you use, you may or may not see hover feedback on tabs. We have the color token `tab.hoverBackground` that allows you to customize it
  - See point 1 about icon hover feedback
3. Valid point, @bpasero I wonder if we should add a command via the context menu for moving the activity bar, similar to the panel?
4. Depending on what OS you are on you will see different menus. Windows and Linux use our custom menus and not native, we have a setting that allows you to change that `window.titleBarStyle`.
5. I am not sure if Visual Studio uses the same Monaco editor that we use, @alexdima might know
6. We have a simplified version of a toolbar, see point number 2 for the screenshot
7. We also have a simplified scrollbar, though they are visually close (Left is VS Code, right is VS):
  ![image](https://user-images.githubusercontent.com/35271042/90577720-785d6a80-e176-11ea-8e3e-068cdbb35ff3.png)
8. It's not perfect but we do have settings that allow you toggle certain features off to increase the editor area. Though VS might be higher density, we have a smaller set of windows/panels that increase the code area:  
  ![image](https://user-images.githubusercontent.com/35271042/90578759-f458b200-e178-11ea-9a60-4454fd367c7f.png)
9. We actually recently introduced [flexible layouts](https://code.visualstudio.com/updates/v1_46#_flexible-layout) where you can move views between the panel, sidebar, and activity bar so this is possible:
  ![flexible layout](https://code.visualstudio.com/assets/updates/1_46/move-views-panel-sidebar.gif)

**Comment by bpasero** [comment]
Yeah I am not sure why we have it not more visible. For example the panel shows this:

![image](https://user-images.githubusercontent.com/900690/90599861-e5641500-e1f5-11ea-8e2e-b038f20ad4a6.png)

@sbatten might know why we are not advertising this more prominently.

**Comment by amervitz** [comment]
> We actually already have an issue that covers the hover cursor issue, i'd encourage you to upvote that one #55346. The biggest change here is that we don't have any hover feedback for icons so this makes it a bit trickier.

Voted, thanks.

> Our tabs are indeed bigger because our toolbar is also on the same line and we need more space to accommodate it.

I would be in favor of separating the toolbar into its own row if that meant the tabs could be smaller 😄 

> Depending on the theme you use, you may or may not see hover feedback on tabs.  We have the color token `tab.hoverBackground` that allows you to customize it

I have tried all of the default included themes, and it appears only the `High Contract` one has any hover feedback. Perhaps some effort can be put into updating them all to include values for the `tab.hoverBackground` color token?

> Depending on what OS you are on you will see different menus. Windows and Linux use our custom menus and not native, we have a setting that allows you to change that `window.titleBarStyle`.

I've switched to the native menus and they're more to my liking, thanks.

> We actually recently introduced flexible layouts where you can move views between the panel, sidebar, and activity bar so this is possible:

This helps, however there appears to be some limitations on which views can be moved (e.g. Extensions can't).

Feel free to decide on the lifetime of this issue as you see fit. Some of these responses with settings have helped, other items seem that finding/creating specific issues and voting on them is best. That said, it would be nice if there was a concerted effort to review how Visual Studio works to identify areas where there are gaps to making vscode look and feel closer to it where there are differences that could be customizable, for example the `window.titleBarStyle` helps a lot. This is the kind of thing that someone would have to be using both of them side by side on a regular basis to recognize and appreciate the differences and how Visual Studio does them differently.
